### PR TITLE
Count issues linked to multiple metrics only once

### LINKED ISSUE #12483: The same issue linked to multiple metric is counted for each metric
**Describe the bug**
When a Jira issue is linked to multiple metrics, the issue summary card counts the issue for each metric.

**To Reproduce**
Steps to reproduce the behaviour:
1. In a report without other linked issues, link the same Jira issue to two metrics
2. Note that the issue summary card says there are two issues

**Expected behaviour**
If the same issue is linked to multiple metrics, it should be counted only once.

**Your environment (please complete the following information):**
- Quality-time: v5.48.1 (but has been present for a long time)

### PR DESCRIPTION
**Pull request by fniessink**
Count a Jira issue linked to multiple metrics only once in the issues dashboard card.

Fixes #12483.

### CODE REVIEW

File: components/frontend/src/dashboard/IssuesCard.test.jsx [hunk 1]
```diff
@@ -9,12 +9,17 @@ const report = {
             metrics: {
                 metric_uuid: {
                     issue_ids: ["ID-1"],
-                    issue_status: [{ status_category: "doing" }],
+                    issue_status: [{ issue_id: "ID-1", status_category: "doing" }],
```
Comment (wkoot) [inline components/frontend/src/dashboard/IssuesCard.test.jsx:12]: So ID-1 is attached to two metrics and only counted as "Doing" once. Is it possible for the same issue ID having multiple statusses; i.e. once "doing" and once "done", when it happened to get resolved in between collector runs for the various metrics? Is that an edge case that we need to catch?

**Review by wkoot — APPROVED** [review]

File: components/frontend/src/dashboard/IssuesCard.test.jsx (hunk 1 above)
Comment (fniessink) [reply components/frontend/src/dashboard/IssuesCard.test.jsx:12]: Yes, that's possible because the issue status is collected, and stored, per metric. To prevent this issue we could store issue status in one `issue_status` list per report instead of a list per metric.  Ticket: https://github.com/ICTU/quality-time/issues/12493
